hutch_python/tests/test_tstpython.py

```diff
@@ -27,16 +30,28 @@ def test_tstpython_scripts():
 
 try:
     import colorama  # noqa
-    has_colorama = True
+
+    # has_colorama = True
+    has_colorama = False
```

Comment:
Was this left in intentionally?

Reply:
🤦  I was clearly not all together when I wrapped this up on Friday